src/WebSdk/ProjectSystem/Targets/Microsoft.NET.Sdk.Web.ProjectSystem.targets

```diff
@@ -65,9 +65,11 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <PropertyGroup>
     <!-- Set the namespaces emitted by the RequestDelegateGenerator for interception when applicable. -->
-    <InterceptorsPreviewNamespaces Condition="'$(EnableRequestDelegateGenerator)' == 'true'">$(InterceptorsPreviewNamespaces);Microsoft.AspNetCore.Http.Generated</InterceptorsPreviewNamespaces>
+    <InterceptorsNamespaces Condition="'$(EnableRequestDelegateGenerator)' == 'true'">$(InterceptorsNamespaces);Microsoft.AspNetCore.Http.Generated</InterceptorsNamespaces>
     <!-- Set the namespaces emitted by the ConfigurationBindingGenerator for interception when applicable. -->
-    <InterceptorsPreviewNamespaces Condition="'$(EnableConfigurationBindingGenerator)' == 'true'">$(InterceptorsPreviewNamespaces);Microsoft.Extensions.Configuration.Binder.SourceGeneration</InterceptorsPreviewNamespaces>
+    <InterceptorsNamespaces Condition="'$(EnableConfigurationBindingGenerator)' == 'true'">$(InterceptorsNamespaces);Microsoft.Extensions.Configuration.Binder.SourceGeneration</InterceptorsNamespaces>
```

Comment:
cc: @RikkiGibson I assume making the change in the .NET 10 SDK is OK or do we expect it to cause issues with older versions of the compiler?

Reply:
If I'm understanding right, we just have to confirm that targeting .NET 8 is not supported with this SDK.

Reply:
If these targets are only used when targeting .NET 10, and older targets are used when targeting .NET 8, then I believe making the change is fine in any case.

Reply:
I believe these targets will kick in for .NET 8 so I suppose it's best to revert this and make the change once .NET 8 goes out of support.